### PR TITLE
Dev depencies updates

### PR DESCRIPTION
**Pull request by lalli-flores**
Includes:
* Latest Node.js version
* Babel
* Nodemon
* Semantic release
* Commitlint
* Husky
* Updates to contribuitng docs

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,10 +3,9 @@
 We'd love for you to contribute to our source code and to make **Webex SDK Component Adapter** even better than it is today!
 If you would like to contribute to this repository by adding features, enhancements or bug fixes, you must follow our process:
 
-  1. [Create an issue](https://github.com/webex/sdk-component-adapter/issues) to propose your solution _before_ you get coding
+  1. [Create an issue](https://github.com/webex/components/issues) to propose your solution _before_ you get coding
```
Comment (adamweeks) [inline CONTRIBUTING.md:6]: Copy pasta fail 😛

**Comment by adamweeks** [comment]
Unit tests are failing to start 😢

File: CONTRIBUTING.md (hunk 1 above)
Comment (lalli-flores) [reply CONTRIBUTING.md:6]: good catch!

**Comment by lalli-flores** [comment]
> Unit tests are failing to start 😢

It's not the tests, it's the linting of "missing dependencies".
Funny because I fixed it in the upcoming PR, but I'll fix it here too

**Review by adamweeks — APPROVED** [review]
LGTM!

**Pull request merged**